internal/pkg/deploy/env.go

```diff
@@ -1,21 +1,21 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Package deploy holds the structures to deploy applications and environments.
+// Package deploy holds the structures to deploy infrastructure resources.
 // This file defines environment deployment resources.
 package deploy
```

Comment:
unrelated: just fyi, my dream state is to completely get rid of all these structs in deploy and have something like this

**hierarchy**
```
deploy/
   cloudformation/
   cdk/
   deploy.go
```

and all these subpackages implement the interface described in deploy.go:
```go
// in deploy.go
type Deployer interface {
    Deploy() error
}
```
for example, in cloudformation/svc.go:
```go
type ServiceDeployer struct {
// service data
}
func (d *ServiceDeployer) Deploy() error { }
```